### PR TITLE
Add deploy workflow - no tests due to hole2 dep

### PR DESCRIPTION
**Pull request by IAlibay**
Quickly adding this here since I did the same for membrane-curvature

### CODE REVIEW

**Review by orbeckst — DISMISSED** [review]
Thanks. Generally looks good but I am blocking this until we have converged on a name #47 .

@lilyminium , as the current person in charge, feel free to dismiss my review at any point.

**Review by lilyminium — APPROVED** [review]
LGTM -- thanks for this @IAlibay!

**Comment by IAlibay** [comment]
@lilyminium I didn't put down an expected publisher on pypi/testpypi, so that'll need to happen before a release

**Pull request merged**
